container/layer.bzl

```diff
@@ -85,6 +86,9 @@ def build_layer(
         "--mode=" + ctx.attr.mode,
     ]
 
+    if toolchain_info.xz_path:
```

Comment:
`and toolchain_info.xz_path != "":`
(as this will be set to empty string if its not found on the path?)

Reply:
just curious, is this a style difference between Python and Starlark? in Python it's common to rely on empty strings being falsy

Reply:
no, its what we need to check for as we're using the template to set the value, and the template includes the "{...}" so we cannot replace its value for None. Its part of our experimenting with toolchain rules + repository rules. You're probably right that this is not idiomatic and we should change how the template creation currently works in the repository rule. We'll make sure to refactor it all homogeneously when we do.

Reply:
ahhh gotcha, thanks!